WolvenKit.App/Controllers/MockGameController.cs

```diff
@@ -24,35 +21,25 @@ public MockGameController()
 
         public IObservable<IChangeSet<RedFileSystemModel>> ConnectHierarchy() => _rootCache.Connect();
 
-        public List<IArchiveManager> GetArchiveManagers(bool loadmods) => new List<IArchiveManager>();
+        public List<IArchiveManager> GetArchiveManagers(bool loadmods) => new();
```

Comment:
I know it is not related to this class or your change, but why does it have `loadmods` parameter? I find it strange that archive manager should do that.